### ISSUE TITLE
HiDPI support

### ISSUE THREAD

**Issue by ja2ui0** [glimpse-editor/Glimpse#293]
One of the greatest UI problems for me, and I'm sure many others, is the ancient design - most grievously the glaring lack of hidpi support in GIMP. I would jump ship to glimpse immediately if that were the only thing that changed. 

In 2020, tons of people are using hidpi (retina / 4k) displays and among them, I would suspect that design and image manipulation folks would represent a hefty number. Having a flagship Linux graphics application with an interface that doesn't respect high resolution seems like a clear oversight which I hope the glimpse project will prioritize and rectify.

**Comment by BrainBlasted** [comment]
This is something we plan for with the move to GTK3. Currently we use GTK2, which does not support HiDPI.On Feb 14, 2020 10:53 AM, Shane Heroux <notifications@github.com> wrote:One of the greatest UI problems for me, and I'm sure many others, is the ancient design - most grievously the glaring lack of hidpi support in GIMP. I would jump ship to glimpse immediately if that were the only thing that changed.
In 2020, tons of people are using hidpi (retina / 4k) displays and among them, I would suspect that design and image manipulation folks would represent a hefty number. Having a flagship Linux graphics application with an interface that doesn't respect high resolution seems like a clear oversight which I hope the glimpse project will prioritize and rectify.

**Comment by ja2ui0** [comment]
What is the timeline for GTK3?

**Comment by BrainBlasted** [comment]
We don't have a defined timeline. It depends largely on when the 
upstream GNU Image Manipulation Program finishes porting to GTK3.

**Comment by TrechNex** [comment]
Correct. Unfortunately we don't have the resources to do the GTK2 to GTK3 port ourselves on a codebase of this size and complexity. We're waiting on upstream to complete that port, which will be present in GNU Image Manipulation Program 3.0.

Upstream has a policy of not giving timelines on any of their releases, so unfortunately we can't give a satisfying answer on when we will be able to re-base on it. Our best guess at this point is "some time in 2021".

We can try to look into things to improve the HiDPi experience in the meantime though. It's possible if we can find someone to do the artwork we could try to provide HiDPi icon and UI themes.

**Comment by TrechNex** [comment]
Also cc @Member1221 as they're running our parallel effort to write a new UI/image editing application. As that would is getting built from scratch we have a lot more flexibility on things like this.

**Comment by Member1221** [comment]
yeah, I would need to get a HiDPI display though as rn I'm using some p crappy second hand 1080p ones.
